### ISSUE TITLE
Harmonize LUT labels between BWH and Chicago user groups

### ISSUE THREAD

**Issue by fedorov** [fedorov/PCampReview#27]
Need to make sure the default LUT currently used contains all the terms needed by the the Chicago group.

**Comment by rweiss42** [comment]
I went ahead and added the color table we came up with to the google doc.  You'll notice when I created this table I spaced out the numbers to allow for future additions and to be sure that similar materials could appear together in the list (avoid how your Tumor_PZ_2 is far from your Tumor_PZ_3).  Do you care that my numbers aren't sequential?  

We could also designate chunks of the number space to different organizations.  BWH could get domain over 1-100, uchicago 100-200, etc?  

As we were discussing last time we spoke, the role of the number itself it less and less important these days since material name is tracked by file name.

**Comment by fedorov** [comment]
for the reference, the LUT is here: http://goo.gl/7Rtgqp

**Comment by fedorov** [comment]
Thank you Robin!

The motivation for this non-consecutive numbering is that our clinician didn't want to have a long list, and have most often used labels on top (not typical to have that many tumors). Also, we are hoping to mark some of the regions automatically (gland sectors), so they are in the middle, and then least likely numbers in the bottom.

Yes, as we discussed, the actual numbers and order are not so important. Material name is.

It's an interesting idea to have chunks of numbers for different organizations, but in a way it's procrastination :) What we need is agree that (for example) cancer_PZ_1 is the same as TumorROI_PZ_1, and get rid of the redundancy. Unless we do that, we won't be able (later on) to store the results in the standard fashion.

**Comment by rweiss42** [comment]
Makes sense.  I just took another look at the list and I think cancer_PZ / TumorROI_PZ are the only place were our lists really overlap so I've changed that.  I'm honestly not the domain expert here so for example I have no idea if *CGTZ would map to our *TZ.

Purely stylistic/non-important question: why do you have the substring ROI scattered around the label names?  Aren't they all ROI's?

**Comment by fedorov** [comment]
CGTZ will not map to TZ .... our clinical user didn't want to break down between the two, since it is hard to differentiate.

Re ROI - you are right, it is redundant, and the names are not consistent in that regard. I missed it and now I have to accept the consequences!

**Comment by fedorov** [comment]
Robin, please let me know when you discussed this with Aytek, and then I will take over.
